build-tools/src/main/resources/repose-style-configs/scalastyle_config.xml

```diff
@@ -93,7 +93,8 @@
         <customMessage>Require space after comma, semicolon, and colon not in string</customMessage>
     </check>
 
-    <check enabled="true" class="org.scalastyle.file.RegexChecker" level="warning">
+    <!-- this causes too many false positives -->
+    <check enabled="false" class="org.scalastyle.file.RegexChecker" level="warning">
```

Comment:
Making sure all `{` weren't starting on lines.
In my specific example, I was using the Scala XML constructs

```scala
<item>
 {IteratorOfElems}
</item>
```
This is completely valid scala code, and the IDE formats it this way. Scalacheck flips out in that rule because the `{` is on a new line